### DISCUSSION TITLE
Why not serde?

### DISCUSSION THREAD

**Discussion by naveedpash** [librasn/rasn#42] [General]
Hey, was just wondering: why did you not implement this as a serde crate?

**Comment by XAMPPRocky** [comment]
Thank you for your question! I'll link to these comments I made on the Rust lang forum that should hopefully explain why this is not `serde`. https://users.rust-lang.org/t/comparison-of-way-too-many-rust-asn-1-der-libraries/58683/9?u=xampprocky

**Reply by naveedpash** [reply]
Thank you, very informative!

Saved me from trying to do this in serde myself
